site/en/docs/apps/analytics/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: "layouts/doc-post.njk"
-title: "Analytics"
+title: "[Chrome Apps] Analytics"
```

Comment:
This should go to seoTitle and not title.

Reply:
I was going by the search results, which seem to pick the title as opposed to the seoTitle. 

I'll switch to seoTitle for now, merge and see what happens... 🧐

Reply:
SERPs use the title, but the index might not have been updated.

Reply:
Eh... the seoTitle already have Chrome Apps on them... and they still show up in search without the Chrome apps label

<img width="442" alt="Screenshot 2023-07-19 at 13 29 00" src="https://github.com/GoogleChrome/developer.chrome.com/assets/37001393/6681f96f-e166-43dc-9d0a-f672069b2001">

![image](https://github.com/GoogleChrome/developer.chrome.com/assets/37001393/2c75401a-5207-406f-91c2-87e34907d10b)

Hmm...

Reply:
🤦 Ian already changed this. The index simply hasn't updated.

Reply:
Sorry for the confusion./